manifests/config.pp

```diff
@@ -2,7 +2,7 @@
 #
 # This class is called from uchiwa
 #
-class uchiwa::config {
+class uchiwa::config inherits uchiwa {
```

Comment:
I'm not sure if inheritance is the best solution for this problem (really, I'm not sure -- not trying to actually say it's not the best solution :)

An alternative solution I came up with was directly in the template:

https://gist.github.com/jtopjian/d2d713db4df12a0df0b3